a/apache-tomcat-native/tomcat-native.sh

```diff
@@ -0,0 +1,57 @@
+# ----------------------------------------------------------------------------
+#
+# Package        : tomcat-native
+# Version        : 1.2.28
+# Source repo    : https://github.com/apache/tomcat-native.git
+# Tested on      : ubi:8.3
+# Script License : Apache License 2.0
+# Maintainer     : Anant Pednekar <Anant.Pednekar@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+#Update Repos
+#yum -y update
+
+#Install Utilities
+yum install -y git
+
+# install Java
+yum install -y java java-devel
+whichJavaString=$(ls /usr/lib/jvm/ | grep -P '^(?=.*java-1.8)(?=.*ppc64le)')
+# Set JAVA_HOME variable 
+export JAVA_HOME=/usr/lib/jvm/$whichJavaString
+whichJavaString=$(ls /usr/lib/jvm/ | grep -P '^(?=.*jre-1.8)(?=.*ppc64le)')
+export JRE_HOME=/usr/lib/jvm/jre
```

Comment:
Should this be like `export JRE_HOME=/usr/lib/jvm/$whichJavaString` or is that correct?
And if that is correct, then what's the use of line 30 `whichJavaString=$(ls /usr/lib/jvm/ | grep -P '^(?=.*jre-1.8)(?=.*ppc64le)')`

Reply:
Yes, `export JRE_HOME=/usr/lib/jvm/$whichJavaString` is correct.